src/main/kotlin/org/oso/core/factories/NotificationFactory.kt

```diff
@@ -35,7 +35,7 @@ class NotificationFactory constructor(
         )
     }
 
-    fun createEmergencyAcceptedPushNotification(to: String, alarmID: Long, helpRequesterId: Long, helpProviderId:  Long) =
+    fun createEmergencyAcceptedPushNotification(to: String, alarmID: String, helpRequesterId: String, helpProviderId: String) =
```

Comment:
As we are using camelcase throughout the application  ``alarmID`` should be renamed to ``alarmId``

Reply:
👍